src/components/TransactionBlock/PlrDaoStakingTransactionBlock.tsx

```diff
@@ -371,13 +381,14 @@ const PlrDaoStakingTransactionBlock = ({
   const getNftList = async () => {
     try {
       if (!accountAddress || !providerAddress || !sdk) return;
-      const output = await sdk.getNftList({
-        account: accountAddress || providerAddress,
-      });
-      let hasNFTContractAddress = output?.items?.filter((nft) => nft.contractAddress === plrDaoMemberNFT.address);
-      if (hasNFTContractAddress?.length) {
-        setIsNFTMember(true);
-      }
+      let accountNFTDetails: NftCollection;
+      let providerNFTDetails: NftCollection;
+      [providerNFTDetails] = await getNftsForChainId(CHAIN_ID.POLYGON, providerAddress, true);
+      [accountNFTDetails] = await getNftsForChainId(CHAIN_ID.POLYGON, accountAddress, true);
```

Comment:
can't u just assign here directly to vars?

Reply:
also what if there are more NFTs than this?

Reply:
it should: run a check on each address if any of holding nfts contain `plrDaoMemberNFT`

Reply:
@poocart refactored the code for fetching NFTs. Please have a look.